src/main/java/org/springframework/hateoas/mediatype/alps/Alps.java

```diff
@@ -38,14 +40,23 @@
  */
 @Value
 @Builder(builderMethodName = "alps")
-@JsonPropertyOrder({"version", "doc", "descriptor"})
+@JsonPropertyOrder({ "version", "doc", "descriptor" })
 @JsonInclude(JsonInclude.Include.NON_NULL)
 public class Alps {
 
-	private final String version = "1.0";
+	private final String version;
 	private final Doc doc;
 	private final List<Descriptor> descriptor;
 
+	@JsonCreator
```

Comment:
Can you elaborate on why we need the creators? So far the classes have been designed to only be used for serialization and object creation is handled via builders.

Reply:
It makes deserialization work which seemed necessary for using WebTestClient. 

What’s wrong with making deserialization operational? We can make that constructor private.

Reply:
I think I had test cases earlier that tapped deserialization, but I clearly changed them to use `jsonPath` in the MockMVC and WebTestClient test cases. I can remove the `@JsonCreator` if you'd like.